webhook/certificates/resources/secret.go

```diff
@@ -27,8 +28,12 @@ import (
 const (
 	// ServerKey is the name of the key associated with the secret's private key.
 	ServerKey = "server-key.pem"
+	// ServerKeyEnv is the env var name for the webhook secret's key eg. `tls.key`.
+	ServerKeyEnv = "KNATIVE_WEBHOOK_SERVER_KEY"
 	// ServerCert is the name of the key associated with the secret's public key.
 	ServerCert = "server-cert.pem"
+	// ServerCertEnv is the env var name for the webhook secret's ca data key eg. `tls.crt`.
+	ServerCertEnv = "KNATIVE_WEBHOOK_SERVER_CERT"
```

Comment:
I realize now the env vars don't influence the certificates & keys being created. Thus I don't think these overrides should be in this package.


This package is for _creating_ and _updating_ certificates

Reply:
Ok will do the webhook options approach thanks.